demos/prefsServerIntegration/schemas/schemas.js

```diff
@@ -0,0 +1,251 @@
+/*
+
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/first-discovery/raw/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery");
+
+    /*******************************************************************************
+     * Auxiliary Schema
+    *******************************************************************************/
+
+    fluid.defaults("gpii.firstDiscovery.auxSchema", {
+        gradeNames: ["fluid.prefs.auxSchema"],
```

Comment:
This file seems to be a copy of https://github.com/fluid-project/first-discovery/blob/master/src/schemas/schemas.js with some modifications to it. Rather than doing this, i think we should just use "gpii.firstDiscovery.auxSchema" as a base grade.